src/core/utils/default_app_categories.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+/** @internal */
+export const DEFAULT_APP_CATEGORIES = Object.freeze({
+  analyze: {
+    label: i18n.translate('core.ui.analyzeNavList.label', {
+      defaultMessage: 'Analyze',
+    }),
+    order: 1000,
+  },
+  observability: {
+    label: i18n.translate('core.ui.observabilityNavList.label', {
+      defaultMessage: 'Observability',
+    }),
+    order: 2000,
+  },
+  security: {
+    label: i18n.translate('core.ui.securityNavList.label', {
+      defaultMessage: 'Security',
+    }),
+    order: 3000,
+  },
+  administration: {
+    label: i18n.translate('core.ui.managementNavList.label', {
+      defaultMessage: 'Administration',
+    }),
+    euiIconType: 'managementApp',
+  },
```

Comment:
for consistency, `administration` should probably also have an `order` defined

Reply:
But why? It is pulled out and rendered separately... Having an `order` defined would be sending a false signal about what's going to happen.